docs/_Interface_IColony.md

```diff
@@ -1376,6 +1397,20 @@ Sets the recipient on an expenditure slot. Can only be called by expenditure own
 |_recipient|address|Address of the recipient
 
 
+### `setExpenditureRecipients`
+
+Sets the recipients in sequential expenditure slots. Can only be called by expenditure owner.
+
+
+**Parameters**
+
+|Name|Type|Description|
+|---|---|---|
+|_id|uint256|Id of the expenditure
+|_firstSlot|uint256|Starting slot to set recipients
+|_recipients|address[]|Addresses of the recipients
+
+
 ### `setExpenditureSkill`
```

Comment:
Same again - if we're introducing `setExpenditureSkills` then I assume we should deprecate `setExpenditureSkill`.